### ISSUE TITLE
训练精度问题

### ISSUE THREAD

**Issue by QMLmulan** [dawn-ech/YOLC#9]
作者您好，在模型训练自己我数据集的时候，在第一个epoch就出现了
2024-08-31 09:14:18,846 - mmdet - INFO - 
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=1000 ] = 0.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=1000 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=1000 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=1000 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=1000 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=300 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=1000 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=1000 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=1000 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=1000 ] = -1.000

2024-08-31 09:14:18,849 - mmdet - INFO -
+-------------------+-------+
| category          | AP    |
+-------------------+-------+
| Offshore-platform | 0.000 |
+-------------------+-------+
2024-08-31 09:14:19,049 - mmdet - INFO - Exp name: yolc.py
2024-08-31 09:14:19,050 - mmdet - INFO - Epoch(val) [1][624]    bbox_mAP: 0.0000, bbox_mAP_50: 0.0000, bbox_mAP_75: 0.0000, bbox_mAP_s: 0.0000, bbox_mAP_m: -1.0000, bbox_mAP_l: -1.0000, bbox_mAP_copypaste: 0.000 0.000 0.000 0.000 -1.000 -1.000
C:\Users\nsd\.conda\envs\yolo\lib\site-packages\mmcv\runner\hooks\logger\text.py:112: DeprecationWarning: an integer is required (got type float).  Implicit conversion to integers using __int__ is deprecated, and may be removed in a future version of Python.
  mem_mb = torch.tensor([mem / (1024 * 1024)],
2024-08-31 09:15:00,117 - mmdet - INFO - Epoch [2][50/2130]     lr: 2.500e-03, eta: 2:12:38, time: 0.821, data_time: 0.231, memory: 4928, loss_center_heatmap: nan, loss_xywh_coarse: nan, loss_xywh_coarse_l1: nan, loss_xywh_refine: nan, loss_xywh_refine_l1: nan, loss: nan, grad_norm: nan
第二个epoch就是nan
2024-08-31 09:21:10,932 - mmdet - INFO - Epoch [2][650/2130]    lr: 2.500e-03, eta: 2:11:30, time: 0.625, data_time: 0.002, memory: 4928, loss_center_heatmap: nan, loss_xywh_coarse: nan, loss_xywh_coarse_l1: nan, loss_xywh_refine: nan, loss_xywh_refine_l1: nan, loss: nan, grad_norm: nan
后面就是一直都是nan，请问这个需要如何解决？

**Comment by mqdigihub** [comment]
可以尝试调小学习率，比如降低到0.001。我训练自己的数据集，也遇到过这种情况，学习率降低到0.001就可以了。 @QMLmulan

**Comment by QMLmulan** [comment]
好的，我尝试一下，谢谢~~




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dawn-ech/YOLC"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月31日(星期六) 中午11:07
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dawn-ech/YOLC] 训练精度问题 (Issue #9)





 
可以尝试调小学习率，比如降低到0.001。我训练自己的数据集，也遇到过这种情况，学习率降低到0.001就可以了。 @QMLmulan
